parser/config_test.go

```diff
@@ -0,0 +1,37 @@
+package parser
+
+import (
+	"context"
+	"testing"
+)
+
+func TestGetConfigurations(t *testing.T) {
+	testTabel := map[string]struct {
+		fileList []string
+	}{
+		"with single type": {
+			[]string{"../examples/kubernetes/service.yaml"},
+		},
+		"with two input with same type": {
+			[]string{"../examples/kubernetes/service.yaml", "../examples/kubernetes/deployment.yaml"},
+		},
+		"with different types": {
+			[]string{
+				"../examples/kubernetes/service.yaml",
+				"../examples/traefik/traefik.toml",
+				"../examples/terraform/gke.tf",
+				"../examples/edn/sample_config.edn",
+			},
+		},
+	}
+
+	for n, test := range testTabel {
+		t.Run(n, func(t *testing.T) {
+			ctx := context.Background()
+			_, err := GetConfigurations(ctx, "", test.fileList)
```

Comment:
I didn't check the configurations because it is hard to look inside. This can be covered in the acceptance tests.